backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/dialog/internal/BpkDialogImpl.kt

```diff
@@ -98,29 +97,61 @@ internal fun BpkFlareDialogImpl(
   }
 }
 
+@Composable
+internal fun BpkImageDialogImpl(
+  onDismissRequest: () -> Unit,
+  title: String,
+  text: String,
+  buttons: List<Dialog.Button>,
+  properties: DialogProperties,
+  textAlign: TextAlign,
+  content: @Composable BoxScope.() -> Unit,
+) {
+  Dialog(onDismissRequest = onDismissRequest, properties = properties) {
+    Surface(
+      modifier = Modifier.padding(top = IconPadding),
```

Comment:
I'm not sure we need to have padding in this case since we don't have icon

Reply:
yeah I checked that with adam when we did the flare dialog - we want the positioning to be consistent and include the padding even if the icon isn't there

Reply:
removed it from both BpkFlareDialog and BpkImageDialog

Reply:
sorry maybe my comment wasn't clear: we DO want the padding

Reply:
oh sorry @marianeum I didn't see your comment. I'll revert it then